src/data/markdown/docs/01 guides/03 Results visualization/09 Amazon CloudWatch.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Amazon CloudWatch"
+excerpt: "The Amazon CloudWatch is a monitoring and observability solution. In this article, we will show you how to send metrics from k6 to Amazon CloudWatch and later visualize them."
+---
+
+k6 can send metrics data to [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/) through the [CloudWatch Agent](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Install-CloudWatch-Agent.html), which is effectively a StatsD integration. These metrics can then be visualized in dashboards.
+
+This guide covers running the CloudWatch integration and visualizing the results:
+
+- Run the CloudWatch agent
+- Run the k6 test
+- Visualize k6 metrics in Amazon CloudWatch
+
+## Prerequisites
+
+- [k6](https://k6.io/open-source), an open-source load testing tool for testing the performance of APIs, microservices, and websites.
+- [Amazon EC2](https://aws.amazon.com/ec2/), an scalable virtual server solution to quickly have an up and running environment for your applications.
+- [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/), a monitoring and observability solution.
+
+## Install k6
+
+For more information about how to install k6, please visit our [installation](https://k6.io/docs/getting-started/installation) guide. For Amazon EC2 instances running Debian 10 (Buster), you also need to install [GnuPG](https://www.gnupg.org/) software to verify repository signature before attempting to install k6 from the official repository:
+
+```bash
+sudo apt install -y gnupg
+```
+
```

Comment:
On the other guides, we don't give instructions to install k6.

It's assumed that you are running k6 in your local machine. Which it makes my next question, why do I need to install k6 in EC2?

I think the installation should go into the details with EC2.

1 - Start with install the CloudWatch agent. (See the instructions)
2 - configure the CloudWatch agent like ...
3 - Start the agent (
4 - Run the test and output the result to the agent.


This is the structure of the instructions for ->  
-> https://k6.io/docs/results-visualization/new-relic
- > https://k6.io/docs/results-visualization/datadog